startPanel.js

```diff
@@ -0,0 +1,85 @@
+class StartPanel extends Panel{
+    constructor(application, db)
+    {
+        super(application, db);
+        this.name = "StartPanel"
+        this.db = db;
+    }
+    show(root)
+    {
+      var template = document.querySelector("#StartPage");
+      var templateClone = document.importNode(template.content, true);
+      var configurationButton = templateClone.querySelector("[data-name='ConfigurationPanelButton']");
+      var gameHandler = templateClone.querySelector("[data-name='GameHandler']");
+      var startGameButton = gameHandler.querySelector("[data-name='StartGame']");
+      var stopGameButton = gameHandler.querySelector("[data-name='StopGame']");
+      var teamA = templateClone.querySelector("[data-name='TeamA']");
+      var teamB = templateClone.querySelector("[data-name='TeamB']");
+      root.appendChild(templateClone);
+      this.setTeamsToDropdown(teamA,teamB);
+      var match = new MatchHandler();
+      configurationButton.addEventListener("click", event =>
+    {
+        this.app.sendAction("ConfigurationPanelRequested");
+        clock = null;   
+    })
+    startGameButton.addEventListener("click", event =>
+    {
+        var gameDuration = gameHandler.querySelector("input").value;
+        var SelectedTeams = this.collectTeams(teamA,teamB);
+        if (this.validateConfigration(gameDuration,SelectedTeams) == true)
```

Comment:
po pierwsze - typo w nazwie :)
po drugie - ważne jest żeby nazwa metody mówiła jakiego rodzau rezultat to zwraca. wartość `true` dla wyrażenia `validateConfiguration` nie daje w żaden sposób do zrozumienia czy `true` oznacza tutaj że jest to poprawna, czy niepoprawna konfiguracja :)

Reply:
Poprawione